airbyte-db/db-lib/src/test/java/io/airbyte/db/mongodb/MongoUtilsTest.java

```diff
@@ -0,0 +1,137 @@
+    assertEquals(objectId, MongoUtils.getBsonValue(BsonType.OBJECT_ID, objectId.toHexString()));
+    assertEquals(new Symbol(value), MongoUtils.getBsonValue(BsonType.SYMBOL, value));
+    assertEquals(new BsonString(value), MongoUtils.getBsonValue(BsonType.STRING, value));
+
+    // Default case
+    assertEquals(value, MongoUtils.getBsonValue(BsonType.MAX_KEY, value));
+
+    // Error case
+    assertEquals(value, MongoUtils.getBsonValue(BsonType.DATE_TIME, value));
+  }
+
+  @Test
+  void testToJsonNodeFromDocument() {
+    final String key = "key";
+    final String value = "foo";
+    final BsonDocument bsonDocument = mock(BsonDocument.class);
+    final Document document = mock(Document.class);
+    final Set<Map.Entry<String, org.bson.BsonValue>> entrySet = Map.of(key, (BsonValue) new BsonString(value)).entrySet();
+
+    when(document.toBsonDocument(any(), any())).thenReturn(bsonDocument);
+    when(bsonDocument.asDocument()).thenReturn(bsonDocument);
+    when(bsonDocument.entrySet()).thenReturn(entrySet);
+
+    final JsonNode jsonNode = MongoUtils.toJsonNode(document, List.of());
+    assertNotNull(jsonNode);
+    assertEquals(value, jsonNode.get(key).asText());
+  }
+
+  @Test
+  void testToJsonNodeFromBsonDocument() {
+    final String key = "key";
+    final String value = "foo";
+    final BsonDocument bsonDocument = mock(BsonDocument.class);
+    final Set<Map.Entry<String, org.bson.BsonValue>> entrySet = Map.of(key, (BsonValue) new BsonString(value)).entrySet();
+
+    when(bsonDocument.asDocument()).thenReturn(bsonDocument);
+    when(bsonDocument.entrySet()).thenReturn(entrySet);
+
+    final JsonNode jsonNode = MongoUtils.toJsonNode(bsonDocument, List.of());
+    assertNotNull(jsonNode);
+    assertEquals(value, jsonNode.get(key).asText());
+  }
+
+  @Test
+  void testTransformToStringIfMarked() {
+    final List<String> columnNames = List.of("_id", "createdAt", "connectedWallets", "connectedAccounts_aibyte_transform");
+    final String fieldName = "connectedAccounts";
+    final JsonNode node = Jsons.deserialize(
+        "{\"_id\":\"12345678as\",\"createdAt\":\"2022-11-11 12:13:14\",\"connectedWallets\":\"wallet1\"," +
+            "\"connectedAccounts\":" +
+            "{\"google\":{\"provider\":\"google\",\"refreshToken\":\"test-rfrsh-google-token-1\",\"accessToken\":\"test-access-google-token-1\",\"expiresAt\":\"2020-09-01T21:07:00Z\",\"createdAt\":\"2020-09-01T20:07:01Z\"},"
+            +
+            "\"figma\":{\"provider\":\"figma\",\"refreshToken\":\"test-rfrsh-figma-token-1\",\"accessToken\":\"test-access-figma-token-1\",\"expiresAt\":\"2020-12-13T22:08:03Z\",\"createdAt\":\"2020-09-14T22:08:03Z\",\"figmaInfo\":{\"teamID\":\"501087711831561793\"}},"
+            +
+            "\"slack\":{\"provider\":\"slack\",\"accessToken\":\"test-access-slack-token-1\",\"createdAt\":\"2020-09-01T20:15:07Z\",\"slackInfo\":{\"userID\":\"UM5AD2YCE\",\"teamID\":\"T2VGY5GH5\"}}}}");
```

Comment:
I know this was copied from the existing test, but we should update this to use a text block
```
"""
{"id":"1234", ...
"""
```